app/lib/actions/katello/host/upload_package_profile.rb

```diff
@@ -0,0 +1,28 @@
+module Actions
+  module Katello
+    module Host
+      class UploadPackageProfile < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        def plan(host, profile)
+          action_subject host
+
+          ::Katello::Pulp::Consumer.new(host.content_aspect.uuid).upload_package_profile(profile) if host.content_aspect.uuid
+          simple_packages = profile.map { |item| ::Katello::Glue::Pulp::SimplePackage.new(item) }
+          host.import_package_profile(simple_packages)
```

Comment:
I think we talked about this, however, need a refresher, the package profile exists without the content aspect correct?